src/index.html

```diff
@@ -11,7 +11,213 @@
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>HTML Form</h1>
+    <form action="https://mate-academy-form-lesson.herokuapp.com/create-application" method="get">
+      <fieldset>
+        <legend>Personal information</legend>
+        <label 
+          for="surename" 
+          class="fieldsetItem"
+        >
+          Surname:
+          <input 
+            type="text" 
+            name="surename" 
+            autocomplete="off"
+            id="surename"
+          >
+        </label>
+        <label 
+          for="name" 
+          class="fieldsetItem"
+        >
+          Name:
+          <input 
+            type="text" 
+            name="name" 
+            autocomplete="off" 
+            required="" 
+            id="name"
+          >
+        </label>
+        <label 
+          for="age" 
+          class="fieldsetItem"
+        >
+          How old are You?
+          <input 
+            type="number" 
+            name="age" 
+            min="1" 
+            max="100" 
+            value="12" 
+            id="age"
+          >
+        </label>
+        <label 
+          for="dateOfbirth" 
+          class="fieldsetItem"
+        >
+          Full date of birth:
+          <input
+            type="date" 
+            name="dateOfbirth" id="dateOfbirth"
+          >
+        </label>
+        <label 
+          for="terms" 
+          class="fieldsetItem"
+        >
+          I accept the term of the agreement
+          <input 
+            type="checkbox" 
+            name="terms" 
+            required="" 
+            id="terms"
+          >
+        </label>
+      </fieldset>
+      <fieldset>
+        <legend>Registration</legend>
+        <label 
+          for="email" 
+          class="fieldsetItem"
+        >
+          E-mail:
+          <input 
+            type="email" 
+            name="email" 
+            placeholder="email@example.com" 
+            required="" 
+            id="email"
+          >
+        </label>
+        <label 
+          for="password" 
+          class="fieldsetItem"
+        >
+          Password:
+          <input 
+            type="password" 
+            name="password" 
+            minlength="8" 
+            maxlength="30" 
+            required="" 
+            id="password"
+          >
+        </label>
+      </fieldset>
+      <fieldset>
+        <legend>An interesting fact about you!</legend>
+        <label class="fieldsetItem">
```

Comment:
This wrapper shouldn't be a label
```suggestion
        <div class="fieldsetItem">
```